app/models/account_filter.rb

```diff
@@ -17,13 +17,13 @@ class AccountFilter
   attr_reader :params
 
   def initialize(params)
-    @params = params
+    @params = params.to_h.symbolize_keys
```

Comment:
The incoming params value here is sometimes a hash and sometimes a strong params object ... this change was added to normalize the strings vs symbols in the keys (some of the controllers which use the AccountFilter to some modification to the keys before using the class, and wind up with a symbols/strings mix).